src/spec/PointerInputSpec.ts

```diff
@@ -0,0 +1,101 @@
+/// <reference path="jasmine.d.ts" />
+
+/// <reference path="Mocks.ts" />
+
+describe('A pointer', () => {
+   
+   var mockWindow = null;
```

Comment:
Since we're running in Phantom, what happens if you simply remove these and reference `window`?